src/main/java/net/pms/network/RequestV2.java

```diff
@@ -241,6 +241,12 @@ public void setTextContent(String content) {
 		this.content = content;
 	}
 
+	/**
+	 * The InputStream for the response to a received request.
+	 * Defining it here make it hopefully automatically closable.
+	 */
+	private InputStream responseInputStream = null;
```

Comment:
I'm not sure about this without testing, since maybe an instance of RequestV2 should be able to send multiple inputStreams. Will have to test